src/main/java/uk/gov/hmcts/reform/professionalapi/controller/external/OrganisationExternalController.java

```diff
@@ -225,6 +228,42 @@
 
     }
 
+    @ApiOperation(
+            value = "Retrieves all Organisations of requested status for HMCTS user with minimal info like organisationIdentifier and its name.",
+            authorizations = {
```

Comment:
It returns only ACTIVE organisations only  right ? can you update the message accordingly